tevfik_aksoy/main.cpp

```diff
@@ -120,28 +154,55 @@ void write_to_file(const GraphPrinter& graph_printer,
   jsonfile.close();
 }
 
+void prepare_temp_directory() {
+  std::filesystem::create_directory("./temp");
+}
+
+void prepare_log_file(Logger& logger) {
+  logger.set_file("./temp/log.txt");
+}
+
 int main() {
   const int threads_count = handle_threads_count_input();
   const int graphs_count = handle_graphs_count_input();
   const int max_depth = handle_depth_input();
   const int new_vertices_num = handle_new_vertices_num_input();
+
   const auto params = GraphGenerator::Params(max_depth, new_vertices_num);
+  auto graphs = std::vector<Graph>();
   auto generation_controller =
       GraphGenerationController(threads_count, graphs_count, params);
+
   auto& logger = Logger::get_instance();
-  std::filesystem::create_directory("./temp");
-  logger.set_file("./temp/log.txt");
-  auto graphs = std::vector<Graph>();
-  graphs.reserve(graphs_count);
 
+  prepare_temp_directory();
+  prepare_log_file(logger);

```

Comment:
Нет смысла разделять логику
```suggestion
  prepare_temp_directory();
  auto& logger = prepare_logger();
```